### DISCUSSION TITLE
On model base of DJL

### DISCUSSION THREAD

**Discussion by Jzow** [deepjavalibrary/djl#540] [Ideas]
I want to know that the DJL team will add OCR model in the next step？ I'm not talking about mnist. I've seen the model library of paddlepaddle that contains OCR models, but when I use the paddle engine, it can't run normally. So I'm wondering if DJL Team train an OCR model and a simple demo through mxnet and tensorflow for the community to learn

**Comment by lanking520** [comment]
I will add an OCR model by the end of this week
